relayer/crates/starknet-integration-tests/src/tests/clear_packets.rs

```diff
@@ -390,15 +377,9 @@ fn test_packet_clearing() -> Result<(), Error> {
                 calldata: call_data,
             };
 
-            let execution = starknet_account_b.execute_v3(vec![call]);
-
-            let tx_hash = execution
-                .send()
-                .await
-                .map_err(<StarknetChain as CanRaiseError<AccountError<SignError>>>::raise_error)?
-                .transaction_hash;
-
-            starknet_chain.poll_tx_response(&tx_hash).await?;
+            let _ = starknet_chain
+                .send_messages_with_signer(wallet_starknet_b, &[StarknetMessage::new(call)])
+                .await?;
```

Comment:
Let's wait for https://github.com/informalsystems/ibc-starknet/pull/359 to merge, which will remove the whole block of calling approve.